guide/english/css/tutorials/perfect-full-page-background-image/index.md

```diff
@@ -2,14 +2,84 @@
 title: Perfect Full Page Background Image
 ---
 ## Perfect Full Page Background Image
+<!-- The article goes here, in GitHub-flavored Markdown. Feel free to add YouTube videos, images, and CodePen/JSBin embeds  -->
+Here is three ways you can achieve the full page background image and I listed the advantage of each solution.
 
-This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/css/tutorials/perfect-full-page-background-image/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
+### Solution 1
 
-<a href='https://github.com/freecodecamp/guides/blob/master/README.md' target='_blank' rel='nofollow'>This quick style guide will help ensure your pull request gets accepted</a>.
+The advantages of this solution:
+
+* fits to specific viewports
+* perfect for hudge banner image
+* still allows you to add content after
+* no whitespace in the viewport.
+
+html:
+
+```
+<div class="bg-image"> </div>
+```
+css:
+
+```
+.bg-image
+{
+  background-size: cover;
+  background-image: url('myimage.jpg');
+  height: 100vh;
+}
+```
+see <a href="https://codepen.io/takachou/pen/pxVPGY/">codepen for solution 1</a>
+
+### Solution 2
+
+This uses the image as a background for the div. This solution allows you to add content down.
+
+html:
+
+```
+<div class="bg-image"> </div>
+```
+
+css:
+
+```
+body, html {
+	height: 100%;
+}
+
+.bg-image {
+    background-image: url("myimage.jpg");
+    height: 100%;
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: cover;
+}
+```
+
+see <a href="https://codepen.io/takachou/pen/gBzRwK">codepen for solution 2</a>
```

Comment:
Please use github markdown style [links](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/docs/style-guide-for-guide-articles.md#links)